package.xml

```diff
@@ -56,6 +56,9 @@
             <file name="zend_abstract_interface/sandbox/php7/sandbox.c" role="src" />
             <file name="zend_abstract_interface/sandbox/php8/sandbox.c" role="src" />
             <file name="zend_abstract_interface/sandbox/sandbox.h" role="src" />
+            <file name="zend_abstract_interface/properties/php7-8/properties.c" role="src" />
```

Comment:
We'll also need to add `zend_abstract_interface/properties/php7-8/properties.c` to `ZAI_SOURCES` for PHP 8 in `config.m4`. And also the new build dirs:

```
PHP_ADD_BUILD_DIR([$ext_builddir/zend_abstract_interface/properties])
PHP_ADD_BUILD_DIR([$ext_builddir/zend_abstract_interface/properties/php7-8])
```

Reply:
I added that in a separate (yet to be committed branch), but will backport it here, makes sense…